packs/equipment-effects/fists-of-the-ruby-phoenix/effect-apricot-of-bestial-might.json

```
@@ -1,10 +1,11 @@
 {
     "_id": "WQ0DxUzMgAvo0zy9",
+    "folder": "xT95plXUpoOOApLs",
     "img": "icons/consumables/fruit/apricot-ripe-orange.webp",
     "name": "Effect: Apricot of Bestial Might",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p><span style=\"font-family:Roboto, sans-serif\">You gain 8 resistance to all physical damage and gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage. </span>You become @UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 1}. If you perform an action with the auditory or linguistic traits, you must succeed at a @Check[flat|dc:5] or the action is lost.</p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Apricot of Bestial Might]</p>\n<p>You gain resistance 8 to all physical damage and gain a tusk unarmed attack with the deadly d12 trait that deals 1d10 piercing damage. You become @UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 1}. If you perform an action with the auditory or linguistic traits, you must succeed at a @Check[flat|dc:5] or the action is lost.</p>"
```
Gotta check for these spans from copy/paste.

Control-shift-v will paste without formatting cruft like that (or copy HTML).

Also the effect doesn't do most of those things, it only adds the strike.